packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts

```diff
@@ -211,7 +211,7 @@ export class Crosshair extends _ModuleSupport.BaseModuleInstance implements _Mod
             const fractionDigits = (axisLayout?.label.fractionDigits ?? 0) + (isInteger(value) ? 0 : 1);
             return formatNumber(value, fractionDigits);
         }
-        return String(value ?? '');
+        return String(value?.toString() ?? '');
```

Comment:
This doesn't make too much sense 😶 
What was the lint error here?
If a change is need then I think you can get rid of the outer `String` (i.e. `return value?.toString() ?? '';`

Reply:
https://typescript-eslint.io/rules/no-base-to-string/

I'm not sure if we should take it tbh, I can undo this change

Reply:
Reverted the fix